enrolments/admin.py

```diff
@@ -1,3 +1,15 @@
 from django.contrib import admin
 
-# Register your models here.
+from .models import Enrolment
+
+
+class EnrolmentAdmin(admin.ModelAdmin):
+    # add class once model is created
+    list_display = ("id", "family")
+
+    ordering = ("-id",)
+
+    search_fields = ("family",)
+
```

Comment:
also couldn't think of any meaningful `list_filter` params either, lmk if you have any ideas 😄